client/components/layout/templates/sidebar.jade

```diff
@@ -2,13 +2,23 @@
   nav#layout-sidebar
     a#layout-sidebar-home( href='/' )
       include ../public/icons/logo.svg
-    a#layout-sidebar-new-article(
-      href='/articles/new'
-      class=(sd.URL == '/articles/new' ? 'is-active' : '')
-    )
-      include ../public/icons/layout_new_article.svg
-      br
-      | New Article
+    if user.isEditorial()
+      #layout-sidebar-new-article(
+        class=(sd.URL == '/articles/new' ? 'is-active' : '')
+      )
+        include ../public/icons/layout_new_article.svg
+        br
+        | New Article
+        nav.layout-sidebar-dropdown
+          a( href='/articles/new?layout=standard' ) Standard Article
+          a( href='/articles/new?layout=classic' ) Classic Article
```

Comment:
I think we should still add feature here, or just remove classic -- trying to avoid any new editorial articles getting the classic layout.

Reply:
Oh haha, whoops I just totally overlooked that. Updating..